.buildkite/scripts/runtask.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cleanup() {
+  echo "trapped exit, cleaning up"
+  task destroy
+  exit $?
+}
+trap cleanup EXIT INT TERM
+task ci-standup TF_CLI_ARGS='-var=tags={
+  "vanta-owner" : "devex"
+  "vanta-non-prod" : "true"
+  "vanta-description" : "cicd-instance-for-devex"
+  "vanta-contains-user-data" : "false"
+  "vanta-user-data-stored" : "none"
+  "vanta-no-alert" : "this-is-a-testing-instance-with-no-stored-data"
+}'
+task test-tls-cluster
```

Comment:
So if I wanted to add a new test, this will be the place where I should put the task execution? (It'll be good to add a guide in CONTRIBUTING.md)

Reply:
I would probably recommend integrating it into the test-tls-cluster task rather than adding a new task (or at least making it a dependency of test-tls-cluster